include/bmi/AbstractCLibBmiAdapter.hpp

```diff
@@ -2,9 +2,8 @@
 #define NGEN_ABSTRACTCLIBBMIADAPTER_HPP
 
 #include <dlfcn.h>
```

Comment:
This `#include` can now be moved down to the .cpp file, since the calls to `dlopen` and `dlsym` are there.